cookbook/upgrade/index.rst

```diff
@@ -0,0 +1,18 @@
+.. index::
+    single: Upgrading
+
+Upgrading
+=========
+
+So a new Symfony release has come out and you want to upgrade, great! Fortunately,
+because Symfony protects backwards-compatibility very closely, this *should*
+be quite easy.
+
+There are three types of upgrades, all needing a little different preparation:
+
+.. toctree::
+    :maxdepth: 2
+
+    /cookbook/upgrade/patch_version
+    /cookbook/upgrade/minor_version
+    /cookbook/upgrade/major_version
```

Comment:
Here is an example of a toctree: http://symfony.com/doc/current/cookbook/security/index.html

And indeed, the text above was just taken from the old article.